specification/sql/resource-manager/readme.md

```diff
@@ -417,6 +417,7 @@ input-file:
 - ./Microsoft.Sql/preview/2021-11-01-preview/DatabaseAdvancedThreatProtectionSettings.json
 - ./Microsoft.Sql/preview/2021-11-01-preview/ServerAdvancedThreatProtectionSettings.json
 - ./Microsoft.Sql/preview/2021-11-01-preview/ManagedServerDnsAliases.json
+- ./Microsoft.Sql/preview/2022-02-01-preview/Databases.json
```

Comment:
if you would like to add a new version replication link, please remove Line 413 and the new version at the bottom.